static/js/components/CourseCard.js

```diff
@@ -63,16 +65,20 @@ const CourseCard = ({
             : null}
         </div>
         <div className="row availability">
-          {availabilityLabel(course.availability)}
+          {availabilityLabel(object.availability)}
         </div>
-        <div className="row platform">
-          <img
-            className="course-platform"
-            src={platformLogoUrls[course.platform]}
-            alt={`logo for ${course.platform}`}
-          />
-        </div>
-        <div className="row price">{minPrice(course)}</div>
+        {objectType === "course" ? (
+          <div className="row platform">
+            <img
+              className="course-platform"
+              // $FlowFixMe: only courses will end up here
+              src={platformLogoUrls[object.platform]}
+              // $FlowFixMe: only courses will end up here
+              alt={`logo for ${object.platform}`}
+            />
+          </div>
+        ) : null}
```

Comment:
fyi in the designs we do want to have the logo show up for bootcamp items here

Reply:
@abdulkdawson can you send me the logo image for bootcamps?

Reply:
sorry - should have clarified, it's already added, if you look at the `platformLogoUrls` object in `constants.js` you can find the URL for it there

Reply:
thanks, added the logo for bootcamps